rafx-framework/Cargo.toml

```diff
@@ -27,6 +29,8 @@ profiling = "0.1.9"
 serde = { version = "1", features = ["serde_derive"] }
 serde_bytes = "0.11"
 glam = { version = "0.13.1", features = [ "serde" ] }
+parking_lot = "0.11.1"
```

Comment:
wondering if we should consistently use parking_lot::Mutex everywhere. (And maybe typedef a RafxMutex so that this is decided upon in one place.)

Reply:
The parking_lot::Mutex is a bit of a double-edged sword if you wanted to typedef it. The problem is that it's more ergonomic to use _but_ that also means it's not "just" a drop-in replacement. You don't need a separate line to declare a guard and you don't need an extra `unwrap`. 

So if we typedef'd it, we'd still need to change every line of code using it when the type under the typedef changed.

Reply:
Yeah I don't think we need to make it swappable with no code changes. It would mainly be to make it clear that this is the "correct" mutex to use in rafx code. We don't need to decide on this now, just a thought.